light_curves/ML_AGNzoo.md

```diff
@@ -786,21 +778,16 @@ plt.legend(fontsize=12)
 plt.axis('off')
 ```
 
-## About this Notebook
-This notebook is created by the IPAC science platform team as a usecase of ML for time domain astrophysics. For questions contact: shemmati@caltech.edu
-
-**Author:** Shoubaneh Hemmati, Research scientist
-
-**Updated On:** 2025-03-31
```

Comment:
Do we drop the last updated dates?

Reply:
Notebooks in this repo haven't historically included a "last updated" date. This one actually had two (one each at the top and bottom) and the scale_up notebook had one but none of the others did. So, not knowing what to put for the others, I opted to drop these ones. But I'm not opposed to adding them back in if we want.

Reply:
Let's wait until we switch to JB2, then this all can be header metadata, along with the authors.